test/scheduling-gates/simple-deployment/00-assert.yaml

```diff
@@ -1,3 +1,11 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+collectors:
+  - type: pod
+    selector: app=test
+  - command: kubectl describe keptnworkloadversion -n $NAMESPACE
```

Comment:
@sudiptob2 Thanks for the contribution! I think the these commands should be also in the other assert file, since  one assertion could pass and the second fail  later

Reply:
also the lifecycle-operator logs are missing, something like :

 kubectl logs -l control-plane=lifecycle-operator -n keptn-system

Reply:
hi, should I run this command directly to the `keptn-system` namespace or in the test namespace using `$NAMESPACE`

Reply:
added. New output when test fails
[test-result.txt](https://github.com/keptn/lifecycle-toolkit/files/13925478/test-result.txt)